addons/common/stringtable.xml

```diff
@@ -676,7 +689,7 @@
             <German>Überprüfe PBOs</German>
             <Czech>Zkontrolovat soubory PBO</Czech>
             <Portuguese>Verificar PBOs</Portuguese>
-            <French>Vérifier les PBOs</French>
+            <!-- <French>Module obsolète - Vérifier les PBOs</French> -->
```

Comment:
I don't see any reason to do this, I think tabler might have problems with xml comments

Reply:
I totally agree with you!
But that's what I was advised yesterday on the slack, but I may have misunderstood.

I would prefer to leave the mention "obsolete", but in uncommented text.
So the next translators won't have to worry about the problem, because you're right, the comments don't appear in tabler.

I'll do that ;).